tyk-pro/scripts/bootstrap.sh

```diff
@@ -22,6 +22,31 @@ syntax_message(){
   exit 1
 }
 
+check_for_existing_organisations(){
+    EXISTING_ORGS=$(CURL_CA_BUNDLE=$CURL_CA_BUNDLE_TYK curl --silent --header "admin-auth: $TYK_ADMIN_SECRET" --header "Content-Type:application/json"  $DASHBOARD_HOSTNAME/admin/organisations 2>&1)
+    OWNER_NAMES=$(echo "$EXISTING_ORGS" | jq -r '.organisations | .[] | .owner_name | @base64')
+    if [[ -z "${OWNER_NAMES}" ]]
+    then
+      echo "No organisation present"
+    else
```

Comment:
We may get rid of `else` statement if we return early after printing echo message.

Reply:
I believe having clarity in the code (especially bash code) is more important than getting rid of a single "else" statement. But I agree  with you that the statement can be deleted if we really wanted to.